prow/plugins/trigger/pr.go

```
@@ -55,21 +55,21 @@ func handlePR(c client, trigger *plugins.Trigger, pr github.PullRequestEvent) er
 		}
 	case github.PullRequestActionReopened:
 		// When a PR is reopened, check that the user is in the org or that an org
-		// member had said "/ok-to-test" before building.
-		comments, err := c.GitHubClient.ListIssueComments(org, repo, num)
+		// member had said "/ok-to-test" before building, resulting in label ok-to-test.
+		labels, err := c.GitHubClient.GetIssueLabels(org, repo, num)
 		if err != nil {
 			return err
 		}
-		trusted, err := trustedPullRequest(c.GitHubClient, trigger, author, org, repo, comments)
+		trusted, err := trustedPullRequest(c.GitHubClient, trigger, author, org, repo, labels)
 		if err != nil {
 			return fmt.Errorf("could not validate PR: %s", err)
 		} else if trusted {
-			err = clearStaleComments(c.GitHubClient, pr.PullRequest, comments)
-			if err != nil {
-				c.Logger.Warnf("Failed to clear stale comments: %v.", err)
+			// Eventually remove need-ok-to-test
+			if github.HasLabel(NeedsOkToTest, labels) {
+				if err := c.GitHubClient.RemoveLabel(org, repo, num, NeedsOkToTest); err != nil {
+					c.Logger.WithError(err).Errorf("Failed at removing %s label", NeedsOkToTest)
```

Please return the error whenever an error is encountered